cookbook/agent_os/dbs/async_postgres_demo.py

```diff
@@ -0,0 +1,44 @@
+"""Example showing how to use AgentOS with a Postgres database, using our async interface"""
+
+from agno.agent import Agent
+from agno.db.async_postgres import AsyncPostgresDb
+from agno.models.openai import OpenAIChat
+from agno.os import AgentOS
+from agno.team.team import Team
+
+# Setup the Postgres database
+db = AsyncPostgresDb(db_url="postgresql+psycopg://ai:ai@localhost:5532/ai")
```

Comment:
should this not be `postgresql+asyncpg://...`? 
can't remember honestly

Reply:
Looks like both `psycopg` and `psycopg_async` work - I'll refactor to `psycopg_async` to make it clear :)